libp2p/switch.nim

```diff
@@ -51,15 +51,15 @@ type
     NoPubSubException* = object of CatchableError
 
     PeerEventKind* {.pure.} = enum
```

Comment:
Sorry, I meant to make this `ConnectionEventKind`, my bad, should have been more explicit.